backend/src/main/java/ai/verta/modeldb/ServiceSet.java

```diff
@@ -111,5 +116,30 @@ private static ArtifactStoreService initializeArtifactStore(
     return artifactStoreService;
   }
 
+  private static void startSpringServer() {
+    var path = System.getProperty(CommonConstants.USER_DIR) + "/" + ModelDBConstants.BACKEND_PID;
+    try {
+      File pidFile = new File(path);
+      if (pidFile.exists()) {
+        try (BufferedReader reader = new BufferedReader(new FileReader(pidFile))) {
+          String pidString = reader.readLine();
+          var pid = Long.parseLong(pidString);
+          var process = ProcessHandle.of(pid);
+          if (process.isPresent()) {
+            LOGGER.warn("Port is already used by modeldb_backend PID: {}", pid);
+            boolean destroyed = process.get().destroy();
+            LOGGER.warn("Process kill for PID `{}` is destroyed: {}", pid, destroyed);
+          }
+        }
```

Comment:
https://docs.oracle.com/javase/9/docs/api/java/util/Optional.html#ifPresent-java.util.function.Consumer-
```suggestion
          ProcessHandle.of(pid).ifPresent(processHandle -> {
            LOGGER.warn("Port is already used by modeldb_backend PID: {}", pid);
            boolean destroyed = processHandle.destroy();
            LOGGER.warn("Process kill for PID `{}` is destroyed: {}", pid, destroyed);            
          });
        }
```

Reply:
Fixed